packages/bierzo-wallet/src/routes/welcome/index.tsx

```diff
@@ -1,30 +1,69 @@
SJtYXNrMzg5MiIKICAgICAgICAgd2lkdGg9IjEwIgogICAgICAgICBoZWlnaHQ9IjkiCiAgICAgICAgIHg9IjE0IgogICAgICAgICB5PSI2IgogICAgICAgICBtYXNrVW5pdHM9InVzZXJTcGFjZU9uVXNlIj4KICAgICAgICA8cGF0aAogICAgICAgICAgIHN0eWxlPSJmaWxsOiNmZmZmZmYiCiAgICAgICAgICAgaW5rc2NhcGU6Y29ubmVjdG9yLWN1cnZhdHVyZT0iMCIKICAgICAgICAgICBkPSJtIDE2LjQxMiwxNC4wMzQgYyAtMS4wNCwtMC43OTggLTEuNjc3LC0yLjA1NiAtMS41OTUsLTMuNDM3IDAuMTMzLC0yLjI1MiAyLjEyMywtMy45NzIgNC40NDQsLTMuODQyIDIuMzIsMC4xMyA0LjA5NCwyLjA2IDMuOTYsNC4zMSBhIDQuMDI2LDQuMDI2IDAgMCAxIC0xLjQ0LDIuODQ0IGwgLTIuNzEsLTMuOTYgeiIKICAgICAgICAgICBpZD0icGF0aDM4OTAiIC8+CiAgICAgIDwvbWFzaz4KICAgICAgPGcKICAgICAgICAgbWFzaz0idXJsKCNjKSIKICAgICAgICAgaWQ9ImczODI3Ij4KICAgICAgICA8cGF0aAogICAgICAgICAgIHN0eWxlPSJjbGlwLXJ1bGU6ZXZlbm9kZDtmaWxsOiM1ZGMwYjc7ZmlsbC1ydWxlOmV2ZW5vZGQiCiAgICAgICAgICAgaW5rc2NhcGU6Y29ubmVjdG9yLWN1cnZhdHVyZT0iMCIKICAgICAgICAgICBkPSJtIDE0LjgwOCwxNC4wMzQgaCA4LjQyMSBWIDYuNzQ3IGggLTguNDIxIHoiCiAgICAgICAgICAgaWQ9InBhdGgzODI1IiAvPgogICAgICA8L2c+CiAgICAgIDxtYXNrCiAgICAgICAgIGlkPSJtYXNrMzg5OCIKICAgICAgICAgd2lkdGg9IjkiCiAgICAgICAgIGhlaWdodD0iMTAiCiAgICAgICAgIHg9IjciCiAgICAgICAgIHk9IjIwIgogICAgICAgICBtYXNrVW5pdHM9InVzZXJTcGFjZU9uVXNlIj4KICAgICAgICA8cGF0aAogICAgICAgICAgIHN0eWxlPSJmaWxsOiNmZmZmZmYiCiAgICAgICAgICAgaW5rc2NhcGU6Y29ubmVjdG9yLWN1cnZhdHVyZT0iMCIKICAgICAgICAgICBkPSJtIDE1LjgxLDI1Ljk1IGMgLTAuMTU3LDEuNDIyIC0wLjkyNywyLjc0MyAtMi4xODMsMy40NzMgLTIuMDQ2LDEuMTkxIC00LjYsMC4zNjQgLTUuNzAxLC0xLjg0NyAtMS4xMDMsLTIuMjEgLTAuMzM3LC00Ljk2OCAxLjcxLC02LjE1OSBhIDMuOTE2LDMuOTE2IDAgMCAxIDMuMjUxLC0wLjMzNCBsIC0yLjA1Nyw0LjgwMyB6IgogICAgICAgICAgIGlkPSJwYXRoMzg5NiIgLz4KICAgICAgPC9tYXNrPgogICAgICA8ZwogICAgICAgICBtYXNrPSJ1cmwoI2QpIgogICAgICAgICBpZD0iZzM4MzQiPgogICAgICAgIDxwYXRoCiAgICAgICAgICAgc3R5bGU9ImNsaXAtcnVsZTpldmVub2RkO2ZpbGw6IzVkYzBiNztmaWxsLXJ1bGU6ZXZlbm9kZCIKICAgICAgICAgICBpbmtzY2FwZTpjb25uZWN0b3ItY3VydmF0dXJlPSIwIgogICAgICAgICAgIGQ9Im0gNy40MjEsMjkuOTcgaCA4LjM5IFYgMjAuODc0IEggNy40MiB2IDkuMDk2IHoiCiAgICAgICAgICAgaWQ9InBhdGgzODMyIiAvPgogICAgICA8L2c+CiAgICAgIDxtYXNrCiAgICAgICAgIGlkPSJtYXNrMzkwNCIKICAgICAgICAgd2lkdGg9IjkiCiAgICAgICAgIGhlaWdodD0iMTAiCiAgICAgICAgIHg9IjIyIgogICAgICAgICB5PSIyMCIKICAgICAgICAgbWFza1VuaXRzPSJ1c2VyU3BhY2VPblVzZSI+CiAgICAgICAgPHBhdGgKICAgICAgICAgICBzdHlsZT0iZmlsbDojZmZmZmZmIgogICAgICAgICAgIGlua3NjYXBlOmNvbm5lY3Rvci1jdXJ2YXR1cmU9IjAiCiAgICAgICAgICAgZD0ibSAyMi4yMDIsMjUuOTk3IDQuOTc3LC0wLjE0OCAtMi4xMjYsLTQuNzY4IGEgMy45MTMsMy45MTMgMCAwIDEgMy4yNTYsMC4yOCBjIDIuMDY0LDEuMTU1IDIuODcsMy45IDEuOCw2LjEyOSAtMS4wNywyLjIzIC0zLjYxMSwzLjEgLTUuNjc1LDEuOTQ0IC0xLjI2NiwtMC43MSAtMi4wNTYsLTIuMDE4IC0yLjIzMiwtMy40MzcgeiIKICAgICAgICAgICBpZD0icGF0aDM5MDIiIC8+CiAgICAgIDwvbWFzaz4KICAgICAgPGcKICAgICAgICAgbWFzaz0idXJsKCNlKSIKICAgICAgICAgaWQ9ImczODQxIj4KICAgICAgICA8cGF0aAogICAgICAgICAgIHN0eWxlPSJjbGlwLXJ1bGU6ZXZlbm9kZDtmaWxsOiM1ZGMwYjc7ZmlsbC1ydWxlOmV2ZW5vZGQiCiAgICAgICAgICAgaW5rc2NhcGU6Y29ubmVjdG9yLWN1cnZhdHVyZT0iMCIKICAgICAgICAgICBkPSJtIDIyLjIwMiwyOS45NDUgaCA4LjM4IHYgLTkuMDk2IGggLTguMzggeiIKICAgICAgICAgICBpZD0icGF0aDM4MzkiIC8+CiAgICAgIDwvZz4KICAgIDwvZz4KICA8L2c+Cjwvc3ZnPgo="
```

Comment:
You can create a file named `./assets/avatar.png`  and then just import it: `import avatar from `./assets/avatar.png`, in this way, when you compile the app, webpack will do some magic and optimize the build.